pkg/apiserver/api_installer.go

```diff
@@ -148,6 +171,19 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage RESTStorage
 
 		itemPath := path + "/{name}"
 		nameParams := append(params, nameParam)
+		namespaceFn = func(req *restful.Request) (namespace string, err error) {
+			return
```

Comment:
Its a named return parameter, which have zero values to start.

However, this made me realize that empty names and namespaces (for namespaced scopes) must be rejected with an error.  Will fix that.

> On Feb 12, 2015, at 5:15 AM, Brian Grant <notifications@github.com> wrote:
> 
> In pkg/apiserver/api_installer.go:
> 
> > @@ -148,6 +171,19 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage RESTStorage
> >  
> >  		itemPath := path + "/{name}"
> >  		nameParams := append(params, nameParam)
> > +		namespaceFn = func(req *restful.Request) (namespace string, err error) {
> > +			return
> I'm still a Go newbie, I guess. What does this do? I don't see a definition of namespace.
> 
> —
> Reply to this email directly or view it on GitHub.
>